ustriage/task.py

```diff
@@ -145,6 +145,39 @@ def short_title(self):
         }[self.obj.target.resource_type_link]
         return ' '.join(self.title.split(' ')[start_field:]).replace('"', '')
 
+    def get_releases(self, open_bug_statuses):
+        """List of chars reflecting per release status.
+
+        Gets a list of chars, one per supported release that show if that task
+        exists (present) and is open (lower case) or closed (upper case).
+
+        Note: This has to stay a fixed length string to maintain the layout
+        """
+        releaseinfo = ''
```

Comment:
We could consider changing this to `release_info for readability`.

Reply:
sure, done